packages/framework/src/Command/Elasticsearch/ElasticsearchIndexesCreateCommand.php

```diff
@@ -7,6 +7,9 @@
 use Shopsys\FrameworkBundle\Component\Elasticsearch\IndexDefinition;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @deprecated This command will be removed in next major version. Use "shopsys:elasticsearch:indexes-migrate" instead.
```

Comment:
Does it make sense to show a deprecation warning in the command itself too?

Reply:
It does. You can use this command directly.